rest-api/tools/deploy_app.sh

```diff
@@ -177,4 +177,12 @@ then
   rm "${tmp_files[@]}"
 fi
 
+test_request=$(curl -s http://localhost:8080/rdr/v1/ | grep version_id)
+if [[ -z "$test_request" ]];
+then
+  echo "${BOLD}Test request failed to return the expected response, something may be wrong with
```

Comment:
We should also update the JIRA ticket if this happens. Ideally we'd send an alert to Slack, too... if you don't want to do that for now, fine by me.